go-controller/pkg/ovn/service.go

```diff
@@ -512,3 +513,79 @@ func svcQualifiesForReject(service *kapi.Service) bool {
 	_, ok := service.Annotations[OvnServiceIdledAt]
 	return !(config.Kubernetes.OVNEmptyLbEvents && ok)
 }
+
+// SVC can be of types 1. clusterIP, 2. NodePort, 3. LoadBalancer,
+// or 4.ExternalName In this scenario we only care about the
+// first 3 since ExternalName means the entities backing the
+// service are external to the cluster and OVN-K dosn't manage them
+func getSvcVips(service *kapi.Service) ([]net.IP, error) {
```

Comment:
on nodeport and external services the packet goes out of the OVN IIRC, so it would not hit the problem with the hairpingtraffic